assets/js/components/AboutPage/AboutPage.test.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { aboutPageFactory } from '@lib/test-utils/factories';
+import AboutPage from './index';
```

Comment:
You could even to do `import AboutPage from './';` or `import AboutPage from `./AboutPage';`